### PR TITLE
Fix a few typos

### PR DESCRIPTION
**Pull request by acuyjet**
Replace `user's` with `users`  
Replace two instances of `it's` with `its`  
Replace `effect` with `affect`

### CODE REVIEW

**Comment by rejas** [comment]
hi @acuyjet could you update your PR to fix the conflicting file?

**Comment by rejas** [comment]
Thx @acuyjet for your patience and work. @patrickkettner could you merge this?

File: docs/intro.md [hunk 1]
```diff
@@ -126,14 +126,15 @@ By default, Modernizr will rewrite `<html class="no-js">` to `<html class="js">`
 
 If you are using a `classPrefix`, such as `supports-`, then you must include that prefix on your `html` element. ie. `supports-no-js` instead of `no-js`.
 
-Finally, if you do not want Modernizr to add any of it's classes, you can set `enableClasses` to `false`. This _does not_ effect the `.no-js` update, so if you do not want that updated either you will need to set `enableJSClass` to `false` in your configuration.
+
+Finally, if you do not want Modernizr to add any of its classes, you can set `enableClasses` to `false`. This is _does not_ affect the `.no-js` update, so if you do not want that updated either you will need to set `enableJSClass` to `false` in your configuration.
```
Comment (rejas) [inline docs/intro.md]: I think that the "is" in "This is _does not_" is too much.

Comment (rejas) [inline docs/intro.md]: I think that the "is" in "This is _does not_" is too much.

Comment (acuyjet) [reply docs/intro.md]: You're absolutely right! Sorry about that.
```suggestion
Finally, if you do not want Modernizr to add any of its classes, you can set `enableClasses` to `false`. This _does not_ affect the `.no-js` update, so if you do not want that updated either you will need to set `enableJSClass` to `false` in your configuration.
```

**Comment by rejas** [comment]
Thx @acuyjet for the quick response, merged!

**Pull request merged**